packages/bridge/src/runtime/app.plugin.mjs

```diff
@@ -41,7 +41,7 @@ export default async (ctx, inject) => {
       provide: inject,
       unmount: () => { },
       use (vuePlugin) {
-        runOnceWith(vuePlugin, () => vuePlugin.install(this))
+        runOnceWith(vuePlugin, () => Vue.use(vuePlugin))
```

Comment:
Is this actually needed? It seems like a correct change to make but want to minimise scope for things breaking.

Reply:
Yes. 
This is because some plugins, such as Vue2ProvideUnheadPlugin, do not have an `install` option.
cc: @danielroe